plugins/chat/assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -92,6 +99,25 @@
   {{#if (or this.chatChannel.isDraft this.chatChannel.isFollowing)}}
     <ChatComposer @draft={{this.draft}} @details={{this.details}} @canInteractWithChat={{this.canInteractWithChat}} @sendMessage={{action "sendMessage"}} @editMessage={{action "editMessage"}} @setReplyTo={{action "setReplyTo"}} @loading={{this.sendingLoading}} @editingMessage={{readonly this.editingMessage}} @onCancelEditing={{this.cancelEditing}} @setInReplyToMsg={{this.setInReplyToMsg}} @onEditLastMessageRequested={{this.editLastMessageRequested}} @onValueChange={{action "composerValueChanged"}} @chatChannel={{this.chatChannel}} />
   {{else}}
-    <ChatChannelPreviewCard @channel={{this.chatChannel}} />
+    {{#if (or this.chatChannel.isDraft this.chatChannel.isFollowing)}}
+      <ChatComposer
+        @draft={{this.draft}}
+        @details={{this.details}}
+        @canInteractWithChat={{this.canInteractWithChat}}
+        @sendMessage={{action "sendMessage"}}
+        @editMessage={{action "editMessage"}}
+        @setReplyTo={{action "setReplyTo"}}
+        @loading={{this.sendingLoading}}
+        @editingMessage={{readonly this.editingMessage}}
+        @onCancelEditing={{this.cancelEditing}}
+        @setInReplyToMsg={{this.setInReplyToMsg}}
+        @onEditLastMessageRequested={{this.editLastMessageRequested}}
+        @onValueChange={{action "composerValueChanged"}}
+        @chatChannel={{this.chatChannel}}
+        @onMentionUpdates={{this.updateMentions}}
```

Comment:
A test is failing with`onMentionUpdates`, I imagine some other component is calling `<ChatComposer />` without having this prop defined

Reply:
`<ChatComposer />` is being rendered twice. Looks like it happened when we resolved the merge conflicts.